pyo3-ffi/src/dictobject.rs

```diff
@@ -96,6 +104,24 @@ pub unsafe fn PyDictViewSet_Check(op: *mut PyObject) -> c_int {
     (PyDictKeys_Check(op) != 0 || PyDictItems_Check(op) != 0) as c_int
 }
 
+#[cfg(not(Py_3_13))]
```

Comment:
The downside of using cfgs this way is that the rendered documentation will declare that it's not available on Python 3.13 and up, which would be confusing.

For that reason it's often better to do something like:
```rust
pub fn foo(){
       #[cfg(a)]{
             ...
       }
      #[cfg(not(a))]{
             ...
       }
}
```

Reply:
I think the way I did this with the `py313_compat` submodule in the new `ffi::compat` module accomplishes this. Let me know if I'm missing something.